internal/pkg/template/env.go

```diff
@@ -112,6 +112,8 @@ type EnvOpts struct {
 	AllowVPCIngress          bool
 	Telemetry                *Telemetry
 
+	SecurityGroupConfig *SecurityGroupConfig
```

Comment:
Can we instead have this under `VPCConfig` and not be a pointer so that it is always set

Reply:
Sure I can have this under `VPCConfig` but the only challenge is that we will need to send empty object from [`getSecurityGroupConfig`](https://github.com/aws/copilot-cli/pull/3810/files#diff-b2b0270288b7891d9a0645336e6267dd39f5b3f4a6b3bf1f6ef19bd4c2806195R201) method as it does not allow us to send nil value in that case which you were recommending [here](https://github.com/aws/copilot-cli/pull/3810#discussion_r929380118)